logger/logger.go

```diff
@@ -113,6 +115,21 @@ func SetAutoFormat() {
 	}
 }
 
+// SetAutoJSONFormat uses the environment variable `LOG_COLOR` and terminal detection to select console or JSON output format
+func SetAutoJSONFormat() {
+	colorYN := strings.ToLower(os.Getenv("LOG_COLOR"))
```

Comment:
What about following https://no-color.org/ instead and having a `NO_COLOR` env var? 

* Default is color (no env var needed)
* When deployed, set a `NO_COLOR` env var which also forces JSON format

Reply:
The default is done by detecting whether stderr is a terminal or not, while the env var can be used to override the behavior (in gitlab CI for example, where color codes are translated to HTML). Normally we don't need to set anything whether on production or for development.

Reply:
OK, sounds fine to me.